sklearn/decomposition/fastica_.py

```diff
@@ -460,9 +475,18 @@ def my_g(x):
     pp. 411-430*
 
     """
-    def __init__(self, n_components=None, algorithm='parallel', whiten=True,
-                 fun='logcosh', fun_args=None, max_iter=200, tol=1e-4,
-                 w_init=None, random_state=None):
+
```

Comment:
please revert this formatting change.

Reply:
Yes, sorry about the formatting. I'll revert all the formatting changes made by `autopep8`